entry_types/scrolled/package/spec/frontend/Video-spec.js

```diff
@@ -1,15 +1,14 @@
 import React from 'react';
 import '@testing-library/jest-dom/extend-expect'
 
-import {renderInEntry} from 'support';
 import {render} from '@testing-library/react'
 
 import {Video} from 'frontend/Video';
 
-describe('Image', () => {
+describe('Video', () => {
   it('renders', () => {
     const {getByRole} =
-      render(<Video id="videoBoatDark" />)
+      render(<Video id="TODO" />)
```

Comment:
CI fails since `render` does not provide the needed React context. Use `renderInEntry` instead.